Ubuntu_14.04/resources/cliConfig/GCL/init.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash -e
```

Comment:
I think we should follow what we are doing https://github.com/Shippable/micro/blob/master/nod/genExec/_common/integrationHandlers/GCL.js#L34 instead of in the new integrations so that we don't end up accidentally breaking something. Eg. We seem to be exporting something `'export %s_INTEGRATION_CREDENTIALFILE_PATH=%s',` and I don't see that here.

Reply:
Hey we need some design changes for exporting these envs which will require changes to reqProc also so I have added a `TO-DO` comment for this and rest everything is working fine.